docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -276,12 +277,15 @@ project settings.
 PSCID is only required if the generation type in the Loris config is set to
 "prompt".
 
-The candidate will be created at the site of the user using the API's site.
+Site is only required if the user using the API is affiliated with multiple sites.
+Otherwise, the candidate will be automatically created at the one site the user 
+using the API is affiliated with.
```

Comment:
I don't think it makes sense for this to be user-variant. Why not just make it required and return an error if they don't have permission for that site?

Reply:
Makes sense to me. I made it behave the way the front-end is behaving right now (which handles the single site and multi site differently, as per your request at the time). But I totally agree it makes more sense to have the site specified in all cases.

Reply:
done